### DISCUSSION TITLE
no puedo subir nada

### DISCUSSION THREAD

**Discussion by batuecas46** [community/community#48076] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The authenticity of host 'github.com (20.201.28.151)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.       
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? no

**Comment by airtower-luna** [comment]
SSH identifies the server you connect to using public key authentication. The first time you connect you need to check if the public key is valid, because SSH can't know. GitHub publishes their SSH public key hashes in the documentation: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints

The Ed25519 fingerprint listed there looks the same as what you posted (but you should check yourself).

**Reply by batuecas46** [reply]
estaba buscando gracias

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
